session/manager.go

```diff
@@ -190,6 +190,11 @@ func (manager *Manager) Start(session *Session, consumerID identity.Identity, co
 		}
 	}()
 
+	log.Info().Msg("Waiting for a first invoice to be paid")
+	if err := engine.WaitFirstInvoice(3 * time.Second); err != nil {
```

Comment:
I would increase first invoice to at least 10 seconds as some consumers can be slow.

Reply:
Left it for 3 seconds for compatibility time only, to not slow down connection from old nodes too much. I'll add TODO for it.